backend/src/main/java/com/example/backend/model/PlayerDTO.java

```diff
@@ -7,13 +7,15 @@
 import lombok.NoArgsConstructor;
 import org.springframework.data.annotation.Id;
 
+
+
 @Builder
 @Data
 @AllArgsConstructor
 @NoArgsConstructor
-public class FancyPartyApp {
+public class PlayerDTO {
 
     @Id
     private String id;
```

Comment:
Wenn wir IdService haben, brauchen wir in PlayerDTO kein Id, oder? Oder sehe ich da etwas falsch?